SPECS-EXTENDED/rubygem-introspection/rubygem-introspection.spec

```diff
@@ -0,0 +1,155 @@
+# Generated from introspection-0.0.2.gem by gem2rpm -*- rpm-spec -*-
+%global gem_name introspection
+
+Name:          rubygem-%{gem_name}
+Version:       0.0.4
+Release:       12%{?dist}
+Summary:       Dynamic inspection of the hierarchy of method definitions on a Ruby object
+# https://github.com/floehopper/introspection/issues/1
```

Comment:
Can be removed - the issue has been resolved in 2011 and version 0.0.4 is from 2016.

Reply:
removed.